actions/check-product-changes/poller.js

```diff
@@ -332,22 +342,48 @@ async function poll(params, aioLibs, logger) {
   checkParams(params);
 
   const {
-    HLX_SITE_NAME: siteName,
-    HLX_PATH_FORMAT: pathFormat,
-    HLX_ORG_NAME: orgName,
-    HLX_CONFIG_NAME: configName,
-    HLX_PRODUCTS_TEMPLATE: productsTemplate,
-    authToken,
+    // required
+    ORG_NAME: orgName,
+    SITE: siteName,
+    PRODUCT_PAGE_URL_FORMAT: pathFormat,
+
+    // optional, fallback to default values
+    CONFIG_NAME: configName = "config",
+    AEM_ADMIN_AUTH_TOKEN: authToken,
+    PRODUCTS_TEMPLATE: productsTemplate,
+    STORE_URL: storeUrl,
+    CONTENT_URL: contentUrl,
+    LOCALES,
+    LOG_LEVEL: logLevel = "info",
+    LOG_INGESTOR_ENDPOINT: logIngestorEndpoint = "https://blazerank-logs-ingestor.adobeaem.workers.dev/api/v1/services/change-detector",
```

Comment:
is this a constant for all customers? can we put it in the env not directly in code?